# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/base/DOMIntersectionObserver.h
# Commit: c428c3ad7725
# Full Hash: c428c3ad7725593f33b4ea2da9851be31be0c1df
# Author: Martin Robinson <mrobinson@igalia.com>
# Date: 2022-11-28 15:41:50
# Regressor Bug: 1791759
# File Overlap Count: 1
# Description:
#   Bug 1791759 - Add support for `content-visibility: auto` r=emilio
#   
#   This change adds support for `content-visibilty: auto` as well as
#   showing and hiding content based on the relevancy of the content as
#   defined in the specification. Changes to relevancy are handled by
# ==============================================================================

diff -r ce553434720c -r c428c3ad7725 dom/base/DOMIntersectionObserver.h
--- a/dom/base/DOMIntersectionObserver.h	Mon Nov 28 09:45:39 2022 +0000
+++ b/dom/base/DOMIntersectionObserver.h	Mon Nov 28 09:53:06 2022 +0000
@@ -163,6 +163,9 @@
   static already_AddRefed<DOMIntersectionObserver>
   CreateLazyLoadObserverViewport(Document&);
 
+  static already_AddRefed<DOMIntersectionObserver>
+  CreateContentVisibilityObserver(Document&);
+
  protected:
   void Connect();
   void QueueIntersectionObserverEntry(Element* aTarget,